fs_storage/models/fs_storage.py

```diff
@@ -163,7 +163,7 @@ def write(self, vals):
     @tools.ormcache()
     def get_id_by_code_map(self):
         """Return a dictionary with the code as key and the id as value."""
-        return {rec.code: rec.id for rec in self.search([])}
+        return {rec.code: rec.id for rec in self.sudo().search([])}
```

Comment:
@adrienpeiffer It's strange to have a `sudo` here but not in others methods.  We should add tests to be sure that we cover all the cases.